test/nyc-integration.js

```diff
@@ -628,55 +622,163 @@ describe('the nyc cli', function () {
       })
 
       it('allows a single file to be instrumented', function (done) {
-        var args = [bin, 'instrument', './half-covered.js', './output']
+        const args = [bin, 'instrument', './half-covered.js', './output']
 
-        var proc = spawn(process.execPath, args, {
+        const proc = spawn(process.execPath, args, {
           cwd: fixturesCLI,
           env: env
         })
 
         proc.on('close', function (code) {
           code.should.equal(0)
-          var files = fs.readdirSync(path.resolve(fixturesCLI, './output'))
+          const files = fs.readdirSync(path.resolve(fixturesCLI, './output'))
           files.length.should.equal(1)
           files.should.include('half-covered.js')
           done()
         })
       })
 
       it('allows a directory of files to be instrumented', function (done) {
-        var args = [bin, 'instrument', './', './output']
+        const args = [bin, 'instrument', './nyc-config-js', './output']
 
-        var proc = spawn(process.execPath, args, {
+        const proc = spawn(process.execPath, args, {
           cwd: fixturesCLI,
           env: env
         })
 
         proc.on('close', function (code) {
           code.should.equal(0)
-          var files = fs.readdirSync(path.resolve(fixturesCLI, './output'))
-          files.should.include('env.js')
-          files.should.include('es6.js')
+          const files = fs.readdirSync(path.resolve(fixturesCLI, './output'))
+          files.should.include('index.js')
+          files.should.include('ignore.js')
+          files.should.include('package.json')
+          files.should.include('node_modules')
           done()
         })
       })
 
       it('allows a sub-directory of files to be instrumented', function (done) {
-        var args = [bin, 'instrument', './subdir/input-dir', './output']
+        const args = [bin, 'instrument', './subdir/input-dir', './output']
 
-        var proc = spawn(process.execPath, args, {
+        const proc = spawn(process.execPath, args, {
           cwd: fixturesCLI,
           env: env
         })
 
         proc.on('close', function (code) {
           code.should.equal(0)
-          var files = fs.readdirSync(path.resolve(fixturesCLI, './output'))
+          const files = fs.readdirSync(path.resolve(fixturesCLI, './output'))
           files.should.include('index.js')
           done()
         })
       })
 
+      it('prevents writing output to a sub-dir of the input dir', function (done) {
```

Comment:
This is potentially a breaking change.  Can we find a way around this?  Maybe something like:
```js
glob.sync(`${input}/*`, {dot:true}).forEach(item => {
  fs.copySync(path.join(input, item), path.join(output, item));
});
```
This might require dealing with directories and files separately, so:
```js
const files = glob.sync(`${input}/*`, {dot: true, nodir: true});
const dirs = glob.sync(`${input}/*/`, {dot: true});
```

Reply:
I'm looking into it now, I should have a solution soon that *maybe* drops the `fs-extra` dependency.

Reply:
Please make sure that whatever solution you choose maintains `chmod` value for the copy.